lib/navigation/segment_navigator.dart

```diff
@@ -66,19 +70,65 @@ class SegmentNavigator extends StatelessWidget {
         "SegmentNavigator: Segment not specified for " + segment.toString());
   }
 
+  /// maps routes to root pages and sub pages
+  Map<String, WidgetBuilder> _routeBuilders(BuildContext context,
+      [Map<String, dynamic> params]) {
+    return {
+      SegmentRoutes.credits: (context) => CreditsPage(),
+
+      SegmentRoutes.help: (context) => HelpPage(
+          onPushHistorySaveState: () => onChangeSegment(AppSegment.history),
+          onPushHistory: () => onChangeSegment(AppSegment.history, true)),
+
+      // TODO add route preferences
```

Comment:
what's up with these 2 TODOs? 
Are they here until we have the corresponding pages?

Reply:
correct, they are for adjusting the other pages later.